src/content/defi/index.md

```diff
@@ -0,0 +1,403 @@
+lang: en
+template: use-cases
+emoji: ":money_with_wings:"
+sidebar: true
+image: ../../assets/finance_transparent.png
+sidebarDepth: 2
+summaryPoints:
+  [
+    "A global, open alternative to the current financial system.",
+    "Products that let you borrow, save, invest, trade, and more.",
+    "Based on open-source technology that anyone can program with.",
+  ]
+---
+
+DeFi is an open and global financial system built for the internet age – an alternative to a system that's opaque, tightly controlled and held together by decades-old infrastructure and processes. It gives you control and visibility over your money. It has given many users exposure to global markets, and alternatives to their local currency or banking options. These "decentralized" services are largely owned by its users and maintained as a public good. It reimagines all of the financial services previously limited to those given permission to access, and opens them up to anyone with an internet connection. So far tens of billions of dollars worth of crypto has flowed through DeFi applications and it's growing every day.
+
+## What's DeFi? {#what-is-defi}
+
+DeFi is a collective term for financial products and services that are accessible to anyone who can use Ethereum – anyone with an internet connection. With DeFi, the markets are always open and there are no centralized authorities who can block payments or deny you access to anything. Services that were previously slow and at risk of human error are automatic and secure now that they're handled by code that anyone can inspect and scrutinize.
+
+There's a booming crypto economy out there, where you can lend, borrow, long/short, earn interest, and more. Crypto-savvy Argentinians have used DeFi to escape crippling inflation. Companies have started streaming their employees their wages in real time. Some folks have even taken out and paid off loans worth millions of dollars without the need for any personal identification.
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/H-O3r2YMWJ4" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## DeFi vs traditional finance {#defi-vs-tradfi}
+
+One of the best ways to see the potential of DeFi is to understand the problems that exist today.
+
+- Some people aren't granted access to set up a bank account or use financial services.
+- Lack of access to financial services can prevent people from being employable.
+- Financial services can block you from getting paid.
+- A hidden charge of financial services is your personal data.
+- Governments and centralized institutions can close down markets at will.
+- Trading hours often limited to business hours of specific time zone.
+- Money transfers can take days due to internal human processes.
+- There's a premium to financial services because intermediray institutions need their cut.
+
+### A comparison {#defi-comparison}
```

Comment:
Came across this Maker talk from a couple years ago on financial reformation.
https://youtu.be/2lxc7qCsvF8?t=1014

I agree with his statement that **transparency** is perhaps the key advantage to DeFi over the traditional system. Real-time audit-ability is a game changer. To paraphrase the talk: any potential customer can look for themselves, see every feature & condition of a protocol or service & verify for themselves if it’s trustworthy. Compare that to a commercial bank.. imagine telling them to "show me your loan book, all assets under management, your risk profile, etc".